app/controllers/util/FormAction.scala

```diff
@@ -32,6 +33,7 @@ object FormAction {
   def bindFromRequest()(implicit request: Request[AnyContent]): FormAction =
     request.body.asFormUrlEncoded.flatMap { body =>
       body.flatMap {
+        case (`addFieldLabel`, values)   => Some(AddField(values.headOption))
```

Comment:
Are we interested in validating if there is more options here - just to make BadRequest ?